apps/docs-snippets/src/guide/types/vector.test.ts

```diff
@@ -78,7 +78,7 @@ describe('Vector Types', () => {
       // #import { arrayify };
       // #context import { BytecodeInputFactory } from '../path/to/typegen';
 
-      const bytecodeAsVecU8 = arrayify(BytecodeInputFactory.bytecode);
+      const bytecodeAsVecU8 = Array.from(arrayify(BytecodeInputFactory.bytecode));
```

Comment:
We should investigate why this is so because the `compute_bytecode_root` expects `bytecodeAsVecU8` which should be acceptable as a `Uint8Array` . Should `BigNumberish` be widened to include `Uint8Array` ? cc @danielbate @nedsalk

Reply:
I believe this is related to:
- https://github.com/FuelLabs/fuels-ts/issues/1553